exchanges/btse/btse_websocket.go

```diff
@@ -242,6 +242,13 @@ func (b *BTSE) wsHandleData(respRaw []byte) error {
 			if err != nil {
 				return err
 			}
+			if amount == 0 {
+				// This occurs when the amount exceeds the decimal returned.
+				// e.g. {"price":"1.37","size":"0.00"} currency: SFI-ETH_0
+				// Opted to not round up to 0.01 as this might skew calculations
+				// more than having it not in the books.
+				continue
+			}
```

Comment:
Since you've got `price` on the other side, I have a feeling we'll be running into this for various shitcoins regardless. So I'm thinking `price == 0 || amount == 0` for both

Reply:
I hope not, because ask side is ascending. Which would then mean the entire bid book will need to be filtered out. 
Will add it though better to purge everything. It's the exchanges problem.

Reply:
Nah that's okay. It was a just in case for something that's not actually occurring. Its probably good to see it start happening in errors so we can go "Hey BTSE, what are you doing?"